### PR TITLE
updates for subprojected wdl4s

### PR DESCRIPTION
**Pull request by mcovarr**
Disclaimer: I didn't actually try to run this, but I did try to fix the things that were obviously affected by the recent subprojecting of wdl4s.

### CODE REVIEW

File: release/release_workflow.wdl [hunk 1]
```diff
@@ -44,7 +44,10 @@ task do_release {
        # Generate new scaladoc
        sbt 'set scalacOptions in (Compile, doc) := List("-skip-packages", "better")' doc
        git checkout gh-pages
-       mv target/scala-2.12/api ${releaseV}
+       for subproj in cwl wdl wom; do
+         mkdir -p ${releaseV}/$subproj
+         mv $subproj/target/scala-2.12/api ${releaseV}/$subproj
+       done
```
Comment (cjllanwarne) [inline release/release_workflow.wdl]: So only the scaladocs need this treatment?

File: release/release_workflow.wdl [hunk 2]
```diff
@@ -259,7 +262,7 @@ workflow release_cromwell {
   call versionPrep as wdl4sPrep { input: 
     organization = organization,
     repo = "wdl4s",
-    file = "build.sbt",
+    file = "project/Version.scala",
```
Comment (cjllanwarne) [inline release/release_workflow.wdl]: Is this not a functional change

**Comment by cjllanwarne** [comment]
ToL: I'm nervous to thumbs up without any way to verify this. On the other hand when build things have gone wrong before it's not been cataclysmic so...

👍 for now... and let the proof of the pudding be in the eating!

File: release/release_workflow.wdl (hunk 2 above)
Comment (mcovarr) [inline release/release_workflow.wdl]: Not sure what you're asking.  This is where the release WDL goes to look for the version number, and when wdl4s was subprojected this is where the version number is now stored.

File: release/release_workflow.wdl (hunk 1 above)
Comment (mcovarr) [inline release/release_workflow.wdl]: AFAICT.  The actual dependency management is handled by sbt.

**Comment by mcovarr** [comment]
Yeah I agree it's not great that I didn't / wasn't able to test this, but I will say it's almost certainly more correct than what was there before given the wdl4s subprojecting.  So it's got that going for it.

**Comment by Horneth** [comment]
👍

**Comment by Horneth** [comment]
👍

**Pull request merged**